Sources/PointFree/Account/Invite.swift

```diff
@@ -13,13 +13,6 @@ import Tuple
 let showInviteMiddleware =
   redirectCurrentSubscribers
     <<< requireTeamInvite
-    <<< filter(
-      validateCurrentUserIsNotInviter,
-      or: redirect(
-        to: .account(.index),
-        headersMiddleware: flash(.warning, "You can’t view your own team invite!")
```

Comment:
now owner's can accept their own invites.

I'm just leveraging the existing invite system for an owner to get back on the team, but in the future we could have a one-click button to re-join the team.